content/en/tracing/faq/trace_sampling_and_storage.md

```diff
@@ -183,7 +183,7 @@ end
 Manually drop a trace:
 
 ```ruby
-Datadog.tracer.trace(name, options) do |span|
+Datadog::Tracing.trace(name, options) do |span|
   # Always Drop the Trace
   span.set_tag(Datadog::Ext::ManualTracing::TAG_DROP, true)
```

Comment:
Should this be `Datadog::Tracing.reject!`?

Reply:
Done!